.github/workflows/bridget_update.yml

```diff
@@ -0,0 +1,35 @@
+name: Create PR in android-news-app when bridget is updated
+
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build and copy bridget.jar
+    runs-on: 4core-ubuntu-latest
+    timeout-minutes: 30
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Setup Gradle
+      uses: gradle/gradle-build-action@v2
+
+    - name: Build jar
+      run: |
+        ./gradlew :library:build
```

Comment:
I recommend saving the jar built by this step using `actions/upload-artifact`, like [in this example](https://github.com/guardian/android-news-app/blob/9f2122b1580f05d7eb1ca7df851c1f0437a28aa0/.github/workflows/build.yml#L66).